finance/server/src/main/java/com/google/finapp/schema_pg.sdl

```diff
@@ -0,0 +1,51 @@
+-- Copyright 2021 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     https://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+CREATE TABLE Account (
```

Comment:
Similar comment around diffing: Would it be possible to branch this from existing schema to make reviews easier ?

Reply:
I can send a diff for this separately, but not through git reviews.

Reply:
sent diff for external review